test/browser_tests/page_objects/page_events.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+function EventsPage() {
+  this.get = function() {
+    browser.get( '/events/' );
+  };
+
+  this.pageTitle = function() { return browser.getTitle(); };
+  this.nav = element( by.css( '.nav-secondary_list' ) );
+  this.heroElem = element( by.css( '.hero' ) );
+  this.hero = {
+    map:     this.heroElem.element( by.css( '.hero_img' ) ),
```

Comment:
Should we just assign this to a variable to avoid doing a lookup? `var _heroElem = this.heroElem = element( by.css( '.hero' ) );`